tests/test_gitAliasTest.py

```diff
@@ -0,0 +1,12 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import subprocess
+from azure.cli.testsdk import ScenarioTest
+
+class TestGitAliasing(ScenarioTest):
+    def test_git_aliases_pr_repo(self):
+        self.cmd('az devops configure --use-git-aliases')
+        repo_help = subprocess.check_output('git repo -h', shell=True)
```

Comment:
What is the difference between `self.cmd` and `subprocess.check_output` ? can we just use self.cmd everywhere?

Reply:
self.cmd is implemented in ScenarioTest and it checks that the command to be executed is starting with 'az' :)